src/components/TodoModal/TodoModal.tsx

```diff
@@ -16,6 +36,7 @@ export const TodoModal: React.FC = () => {
               data-cy="modal-header"
             >
               Todo #2
```

Comment:
remove it

Reply:
Hi! What exactly do I need to remove here, bit confused.